docs/docfx.json

```diff
@@ -1158,6 +1158,9 @@
         "docs-ref-autogen/word*/**": [
           "https://authoring-docs-microsoft.poolparty.biz/devrel/7e593268-8d01-46ef-aa78-9d1388e6a294"
         ]
+      },
+      "ms.topic": {
```

Comment:
There's already a global metadata section with the value `"ms.topic": "reference"`. How do these two interact?